### ISSUE TITLE
Dependabot PR trigger a false positive

### ISSUE THREAD

**Issue by Th3S4mur41** [github/accessibility-alt-text-bot#28]
[GitHub's accessibility alt text bot](https://github.com/github/accessibility-alt-text-bot) is complaining about images missing alt text in dependabot PR

> Uh oh! https://github.com/dependabot[bot], the image you shared is missing helpful alt text. Check your pull request body.
> 
> Alt text is an invisible description that helps screen readers describe images to blind or low-vision users. If you are using markdown to display images, add your alt text inside the brackets of the markdown image.
> 
> Learn more about alt text at [Basic writing and formatting syntax: images on GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images).

![dependabot PR with accessibility bot error](https://github.com/dependabot/dependabot-core/assets/5543595/09da3eb1-aab7-4244-9bb3-d6caea830523)

The only image in the PR is the compatibility score which does have an alt text

```md
[![Dependabot compatibility score](https://dependabot-badges.githubapp.com/badges/compatibility_score?dependency-name=github/accessibility-alt-text-bot&package-manager=github_actions&previous-version=1.2.0&new-version=1.3.0)](https://docs.github.com/en/github/managing-security-vulnerabilities/about-dependabot-security-updates#about-compatibility-scores)
```

Here the HTML output as well to verify:
```html
<a href="https://docs.github.com/en/github/managing-security-vulnerabilities/about-dependabot-security-updates#about-compatibility-scores"><img src="https://camo.githubusercontent.com/dcc310bf5bed38d53e532c75b4c98d348b39d0121dab9d3bc931d10b10d7614b/68747470733a2f2f646570656e6461626f742d6261646765732e6769746875626170702e636f6d2f6261646765732f636f6d7061746962696c6974795f73636f72653f646570656e64656e63792d6e616d653d6769746875622f6163636573736962696c6974792d616c742d746578742d626f74267061636b6167652d6d616e616765723d6769746875625f616374696f6e732670726576696f75732d76657273696f6e3d312e322e30266e65772d76657273696f6e3d312e332e30" alt="Dependabot compatibility score" data-canonical-src="https://dependabot-badges.githubapp.com/badges/compatibility_score?dependency-name=github/accessibility-alt-text-bot&amp;package-manager=github_actions&amp;previous-version=1.2.0&amp;new-version=1.3.0" style="max-width: 100%;"></a>
```

**Comment by kendallgassner** [comment]
related to #30
